doc/global/water/index.rst

```diff
@@ -1,71 +1,213 @@
+
+Energy-Water-Land Linkages
+==========================
+
+MESSAGEix-Nexus captures multiple nexus interactions:
+
+**Energy → Water**: 
+
+* Cooling water requirements for thermal power plants (see :doc:`cooling` for detailed cooling technology representation)
+* Water consumption in fuel extraction (coal mining, unconventional oil and gas)
+* Hydropower production from surface water resources
+* Energy requirements for water supply (pumping, treatment, desalination)
+
+**Water → Energy**:
+
+* Water availability constraints on thermal power plant siting and operation
+* Cooling technology choices driven by water scarcity
+* Hydropower generation governed by river flows and reservoir storage
+* Groundwater pumping costs dependent on aquifer depth
+
+**Land → Water** (via GLOBIOM linkage):
+
+* Irrigation water demands for crop production
+* Land use change impacts on runoff and water availability
+* Water allocation between agriculture and other sectors
+
+**Climate → Water-Energy-Land**:
+
+* Temperature and precipitation changes affecting water availability
+* Extreme events (droughts, floods) impacting all sectors
+* Climate-driven changes in cooling water requirements and efficiency
+* Shifts in crop water demands and irrigation needs
+
+Climate Change Impacts
+======================
+
+The nexus module incorporates climate change impacts on both water availability and energy systems (Awais et al., 2024 :cite:`awais_2024_nexus`):
+
+**Hydrological Impacts**: Basin-specific changes in renewable water availability derived from global hydrological models (PCR-GLOBWB, CWatM) forced by climate model outputs. Impacts include:
+
+* Changes in mean annual runoff and groundwater recharge
+* Shifts in seasonal water availability patterns
+* Increased frequency and severity of droughts
+* Modified snowmelt timing in snow-dominated basins
+
+**Energy System Impacts**:
+
+* Reduced thermal power plant efficiency due to higher ambient temperatures
+* Increased cooling water requirements from higher water temperatures
+* Changes in hydropower generation potential
+* Modified electricity demand patterns (cooling vs. heating)
+
+**Adaptation Measures**: The model can endogenously select adaptation measures such as:
+
+* Shifts to less water-intensive cooling technologies
+* Investment in desalination and water reuse
+* Inter-basin water transfers
+* Changes in electricity generation technology mix
+
+For comprehensive discussion of climate change impacts on the water-energy nexus, see :doc:`climate_impacts`.
+
+Sustainable Development Goals
+==============================
+
+The nexus module includes optional constraints to represent progress toward water-related Sustainable Development Goals (Awais et al., 2024 :cite:`awais_2024_nexus`):
+
+**SDG 6**: Clean water and sanitation
+
+* Targets for urban and rural water access rates
+* Wastewater treatment coverage requirements
+* Water use efficiency improvements
+
+**SDG 7**: Affordable and clean energy
```

Comment:
Maybe we should remove SDG7 as it is not in the water module. it was just in some NAVIGATE branches, but never migrated, and now obsolete